tests/mocha/toolbox_test.js

```diff
@@ -4,7 +4,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-suite('Toolbox', function() {
+suite.only('Toolbox', function() {
+  function convertToolBoxToJSONCaptureWarnings(toolboxXml) {
+    // TODO(#3985): Remove after warning log has been removed.
+    var toolboxJson;
+    captureWarnings(() => {
```

Comment:
Do you have a way to make this only capture warnings matching a specific message (or with a specific prefix)? Capturing all warnings seems like a way to miss real warnings.

Reply:
I wanted to be more lenient so that that tests wouldn't fail after warning log was removed, but I can check the number of warnings and optionally use regex. For other tests I haven't been using regex to check the content of the warnings, but that can be changed (those that are capturing are only checking the count of warnings).

Reply:
I don't think you need to check the number of warnings, just that if the warning doesn't match a regex you call `nativeConsoleWarn` to pass it through.

Reply:
I found that the warning is not always logged. I added a check to only allow up to 1 warning, and to check the contents of that warning message. Since in working on this I created a helper for asserting a specific warning was logged, that wasn't used, I'll add support for that in a separate PR (this is a special case, but in other cases we always expect a warning).